### ISSUE TITLE
Next release number?

### ISSUE THREAD

**Issue by mtb-za** [agile-geoscience/striplog#133]
Some of the changes are to deprecate some of the existing args in favour of those used by `genfromtxt`, along with removing a couple that are no longer needed. The current release is 0.8.8. Should we plan for these to be released as 0.9, 0.8.9, or 0.8.10? Are we planning a 0.8.9 release at all?

I feel like we might have made enough improvements to warrant a 0.9 if we finish cleaning up (most of) the IO things (`from_csv`, `from_text`, `to_json` and a couple of others) that are currently in the T21 project list, along with some of the bugs that we have been discussing, notably #127.

Consensus?

**Comment by kwinkunks** [comment]
Agree. Since we're changing things pretty substantially, we should go to 0.9 next.

**Comment by kwinkunks** [comment]
Where are those functions at now, @mtb-za ?

I don't think they are in production yet, they maybe just need tests?

**Comment by mtb-za** [comment]
They are on a branch, which I am planning to take a proper look at tomorrow. I think that `to_csv` is probably ready, but needs tests. Less sure on the others. That one uses numpy's `loadtxt` internally, passing anything extra as kwargs to that.
